charts/linkerd2/values.yaml

```diff
@@ -210,14 +252,22 @@ smiMetrics:
   # if empty, Helm will auto-generate this field, unless externalSecret is set to true.
   caBundle: |
 
+# set resources for smi-metrics and its linkerd proxy respectively
+# see global.proxy.resources for details.
+#smiMetricsResources:
+#smiMetricsProxyResources:
+
 # Configuration for Add-ons
 
 grafana:
   enabled: true
   name: linkerd-grafana
   image:
     name: gcr.io/linkerd-io/grafana
+  # set resource requests and limits for grafana and its linkerd proxy respectively
+  # see global.proxy.resources in the linkerd2 chart for details.
   # resources:
+  #proxy.resources:
```

Comment:
Please remove the `#resources` line and replace `#proxy.resources` with
```
#proxy
#  resources:
```
If we use `proxy.resources`  in a single line it doesn't work, according to my tests.

Reply:
Would you please address this before this gets merged? :slightly_smiling_face:

Reply:
done